packages/wrangler/e2e/c3-integration.test.ts

```diff
@@ -68,6 +68,7 @@ describe("c3 integration", () => {
 			Published tmp-e2e-wrangler (TIMINGS)
 			  https://tmp-e2e-wrangler.SUBDOMAIN.workers.dev
 			Current Deployment ID: 00000000-0000-0000-0000-000000000000
+			Current Version ID: 00000000-0000-0000-0000-000000000000
 			NOTE: "Deployment ID" in this output will be changed to "Version ID" in a future version of Wrangler. To learn more visit: https://developers.cloudflare.com/workers/configuration/versions-and-deployments"
```

Comment:
@WalshyDev can we also update the copy on the "NOTE". Sorry if the change wasn't super clear on the ticket. 

`Note: Deployment ID has been renamed to Version ID. Deployment ID is present to maintain compatibility with the previous behavior of this command. This output will change in a future version of Wrangler. To learn more visit: https://developers.cloudflare.com/workers/configuration/versions-and-deployments.`